Packages/com.unity.inputsystem/Documentation~/Layouts.md

```diff
@@ -1,12 +1,12 @@
->__Note__: Layouts are an advanced, mostly internal feature of the Input System. You don't need to understand them to use the Input System. Knowledge of the layout system is mostly useful if you want to support custom Devices or change the behavior of existing Devices.
-
 # Layouts
 
-A layout describes a memory format for input, and the Input Controls to build in order to read and write data to or from that memory.
-
 Layouts are the central mechanism by which the Input System learns about types of Input Devices and Input Controls. Each layout represents a specific composition of Input Controls. By matching the description of a Device to a layout, the Input System is able to create the correct type of Device and interpret the incoming input data correctly.
 
-The Input System ships with a large set of layouts for common Control types, as well as for common Devices. For other Device types, the system automatically generates layouts on the fly based on the Device description that the Device's interface reports.
+>__Note__: Layouts are an advanced, mostly internal feature of the Input System. You don't need to understand them to use the Input System. Knowledge of the layout system is mostly useful if you want to support custom Devices or change the behavior of existing Devices.
```

Comment:
As discussed on Slack, I think it's OK to remove the second sentence from this note.